functions/Start-DbaXESession.ps1

```diff
@@ -0,0 +1,133 @@
+function Start-DbaXESession {
+<#
+	.SYNOPSIS
+	Starts Extended Events sessions.
+
+	.DESCRIPTION
+	This script starts Extended Events sessions on a SQL Server instance.
+
+	.PARAMETER SqlInstance
+	The SQL Instances that you're connecting to.
+
+	.PARAMETER SqlCredential
+	Credential object used to connect to the SQL Server as a different user
+
+	.PARAMETER Session
+	Only start specific Extended Events sessions.
+
+	.PARAMETER AllSessions
+	Start all Extended Events sessions on an instance, ignoring the packaged sessions: AlwaysOn_health, system_health, telemetry_xevents.
+
+	.PARAMETER Silent
+	If this switch is enabled, the internal messaging functions will be silenced.
+
+	.NOTES
+	Tags: Memory
+	Author: Doug Meyers
+	Website: https://dbatools.io
+	Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+	License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+	.LINK
+	https://dbatools.io/Start-DbaXESession
+
+	.EXAMPLE
+	Start-DbaXESession -SqlInstance sqlserver2012 -AllSessions
+
+	Starts all Extended Event Session on the sqlserver2014 instance.
+
+	.EXAMPLE
+	Start-DbaXESession -SqlInstance sqlserver2012 -Session xesession1,xesession2
+
+	Starts the xesession1 and xesession2 Extended Event sessions.
+
+	.EXAMPLE
+	Get-DbaXESession -SqlInstance sqlserver2012 -Session xesession1 | Start-DbaXESession
+
+	Starts the sessions returned from the Get-DbaXESession function.
+
+#>
+	[CmdletBinding(DefaultParameterSetName='Session')]  
+	param (
+		[parameter(Position=1, Mandatory, ParameterSetName='Session')]
+		[parameter(Position=1, Mandatory, ParameterSetName='All')]
+		[Alias("ServerInstance", "SqlServer")]
+		[DbaInstanceParameter[]]$SqlInstance,
+
+		[parameter(ParameterSetName='Session')]
+		[parameter(ParameterSetName='All')]
+		[PSCredential]$SqlCredential,
+
+		[parameter(Mandatory, ParameterSetName='Session')]
+		[Alias("Sessions")]
+		[object[]]$Session,
+
+		[parameter(Mandatory, ParameterSetName='All')]
+		[switch]$AllSessions,
+
+		[parameter(Mandatory, ValueFromPipeline, ParameterSetName='Object')]
+		[object]$SessionCollection,
+
+		[switch]$Silent
+	)
+	
+	begin {
+		if ([System.Reflection.Assembly]::LoadWithPartialName("Microsoft.SqlServer.Management.XEvent") -eq $null) {
+			Stop-Function -Message "SMO version is too old. To collect Extended Events, you must have SQL Server Management Studio 2012 or higher installed."
+		}
+
+		# Start each XESession
+		function Start-XESessions {
+			[CmdletBinding()]
+			param ([object]$xesessions)
+
+			foreach ($x in $xesessions) {
```

Comment:
Please adjust this to be `$xe`....a bit more understandable compared to just "x".

As well, on variables that are two words we have been using camelCase. So `$xesessions` would be `$xeSessions`...we can clean this up after but if you can do it now would save us some time 💯